exercises/es/exc_02_14.py

```diff
@@ -1,11 +1,11 @@
 import json
-from spacy.lang.en import English
+from spacy.lang.es import Spanish
 
-with open("exercises/en/countries.json") as f:
+with open("exercises/es/countries.json", encoding="utf8") as f:
```

Comment:
Note to self: we should backport the `encoding` to all other languages.

Reply:
Yes! This has been an issue for running these exercises.